docs/JLBP-0022.md

```diff
@@ -0,0 +1,56 @@
+---
+jlbp:
+  id: JLBP-22
+permalink: /JLBP-22
+---
+
+# Declare all dependencies
+
+If your code references a class—for example, by invoking a method in that class—declare a dependency that includes that class in your pom.xml, build.gradle,
```

Comment:
The varying line lengths make this hard to review. Can you run mdformat?

Reply:
-bash: mdformat: command not found